### ISSUE TITLE
'new Something()' produces undefined childNode, throws exception

### ISSUE THREAD

**Issue by panta82** [squarewolf/node-neuter#4]
I'm not sure if this is a recent regression or if I'm just crazy. It seems any `new Something()` construct chokes up the scanner.

Reproduce steps:

1) Create a file 'broken.js' with content:
```javascript
var t = new Date();
```

2) Create file 'test.js' with content:
```javascript
var Neuter = require("neuter"),
	neuter = new Neuter();

neuter.parse('broken.js', function(err, sourceNode) {});
```

3) From terminal, execute
```
npm install neuter
node test.js
```

This should throw the error:
```
TypeError: Cannot read property 'type' of undefined
    at Object._onImmediate (/home/ivan/dev/node/neuter-test/node_modules/neuter/lib/scanner.js:213:18)
    at processImmediate [as _immediateCallback] (timers.js:330:15)
```

Tested on debian 64-bit, node 0.10.17, neuter version 0.1.2.

**Comment by tvervest** [comment]
Hey guys,

First of all sorry for the lack of support in the library!

I've just patched the issue in #6 after @jorickw kept nagging (he sits next to me at the office, so no getting away from him). I'll be releasing a new version of node-neuter shortly which resolves the problem.

**Comment by tvervest** [comment]
New version (0.1.3) is public and should resolve this issue
